app/components/Blockchain/Asset.jsx

```diff
@@ -503,14 +503,15 @@ class Asset extends React.Component {
                     </th>
                     <th> <Translate content="explorer.asset.price_feed_data.maintenance_collateral_ratio"/> </th>
                     <th> <Translate content="explorer.asset.price_feed_data.maximum_short_squeeze_ratio"/> </th>
-                    <th> <Translate content="explorer.asset.price_feed_data.published"/> </th>
+                    <th> <Translate content="explorer.asset.price_feed_data.published"/> </th> 
                 </tr>
             </thead>
         )
         for (var i = 0; i < feeds.length; i++) {
             var feed = feeds[i];
             var publisher = feed[0];
             var publishDate = new Date(feed[1][0]);
+            publishDate.setTime(publishDate.getTime() - publishDate.getTimezoneOffset()*60*1000);
```

Comment:
Different browsers will treat this differently so it won't work as expected in firefox for example.